docker/frontend/dev.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:20.11.1-alpine3.19@sha256:c0a3badbd8a0a760de903e00cedbca94588e609299820557e72cba2a53dbaa2c as build
```

Comment:
This file shouldn't be here. It shouldn't be needed anyway. `docker compose build` / `docker compose up` starts the frontend in development mode with hot reloading.